<launch>
	<rosparam command="load" file="$(find wall_follow)/launch/wallFollow_params.yaml"/>
	<!--node pkg="wall_follow" type="wall_follow_pc_subscriber" name="pcs1"/-->
</launch>
